edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest/OptionAssertions.cs

```diff
@@ -0,0 +1,137 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Agent.Kubernetes.IntegrationTest
+{
+    using System.Diagnostics.Contracts;
+    using FluentAssertions;
+    using FluentAssertions.Execution;
+    using FluentAssertions.Primitives;
+    using Microsoft.Azure.Devices.Edge.Util;
+
+    public static class OptionAssertionExtensions
```

Comment:
>OptionAssertionExtensions [](start = 24, length = 25)

Could we move this to Microsoft.Azure.Devices.Edge.Util.Test?  Seems like this is useful beyond k8s.

Reply:
Yes, it can be reused somewhere else. But it requires `FluentAssertions` to be installed for project and only `Kubernetes.IntegrationTests` only project which uses it.